management-app/.gitignore

```diff
@@ -0,0 +1,25 @@
+/target/
```

Comment:
This settings can be included in the global `.gitignore` file so each Maven submodule can reuse them.